tensorflow/contrib/layers/python/layers/layers.py

```diff
@@ -319,9 +319,10 @@ def _fused_batch_norm(
                        (inputs.name, params_shape))
```

Comment:
Remove the TODO(reedwm) above. Github doesn't let me comment on a line far away from a change. (Any idea if doing so is possible?)

Reply:
Done